mods/ra/chrome/ingame-observer.yaml

```diff
@@ -124,13 +124,14 @@ Container@OBSERVER_WIDGETS:
 			ClickThrough: false
 			Visible: false
 			Children:
+				LogicKeyListener@REPLAY_KEYHANDLER:
 				Button@BUTTON_PAUSE:
 					X: 9
 					Y: 5
 					Width: 28
 					Height: 28
 					Background: sidebar-button-observer
-					TooltipText: Pause
+					TooltipText: Pause (Spacebar)
```

Comment:
The tooltip will automatically show the hotkeys (in a different color) after you set up the button key bindings, so these changes will be able to go too.